oeps/oep-0002.rst

```diff
@@ -0,0 +1,101 @@
+==========================
+OEP-2: Repository Metadata
+==========================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`oep-2`                              |
++---------------+-------------------------------------------+
+| Title         | Repository Metadata                       |
++---------------+-------------------------------------------+
+| Last-Modified | 2016-04-21                                |
++---------------+-------------------------------------------+
+| Author        | Calen Pennington <cale@edx.org>           |
++---------------+-------------------------------------------+
+| Arbiter       |                                           |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Best Practice                             |
++---------------+-------------------------------------------+
+| Created       | 2016-04-21                                |
++---------------+-------------------------------------------+
+
+Abstract
+========
+
+Proposes a standard format for repository metadata for Open edX projects.
+
+Motivation
+==========
+
+One of the primary goals for Best Practice OEPs is to standardize techniques
+and tools across Open edX repositories. However, without
+tooling, it will be difficult to move all of the repositories in the same
+direction (and hard to validate which ones have been updated). This OEP exists
+to fill that gap.  By providing a standard format for metadata about accordance
+to Best Practice OEPs, tools can be developed in the future to report on the
+current status of all of the Open edX repositories, and how well they follow
+the Best Practices listed in the OEPs.
+
+Bear in mind that particular classes of Open edX repositories may have
+different Best Practices. The practices that apply to an XBlock may not be the
+same as those that apply to a standalone Django service or to a Django app.
+
+Specification
+=============
+
+Each repo in the Open edX system will include a file ``openedx.yml``, with the
+following fields:
+
+``owner``: string
+    The github user/team responsible for the repository.
+
+``nick``: string
+    A short-name for the repository used by reporting tools.
+
+``area``: string
```

Comment:
@cpennington: this might be more flexible as `tags`, which could be a list of strings, and would better handle the case where a repository covers multiple "areas." What do you think?